.github/docker/centreon-web/alma8/slim-configuration/exec.txt

```diff
@@ -11,18 +10,13 @@ su - centreon-gorgone -s /bin/bash -c "sudo service centengine reload"
 API_TOKEN=$(curl -s -d '{"security": {"credentials": {"login": "admin","password": "Centreon!2021"}}}' -H "Content-Type: application/json" -X POST http://127.0.0.1/centreon/api/latest/login | jq -r .security.token)
 curl -s -d '{"check":{"is_forced":true},"resources":[{"id":26,"parent":{"id":14},"type":"service"},{"id":14,"parent":null,"type":"host"}]}' -H "X-AUTH-TOKEN:$API_TOKEN" -H "Content-Type: application/json" -X POST 'http://127.0.0.1/centreon/api/latest/monitoring/resources/check'
 
-sleep 5
+dnf search vim
```

Comment:
is it useful ?

Reply:
Yes on order to détecte dnf usage by slimtoolkit via syscall